core/src/service/usage_service.rs

```diff
@@ -22,22 +21,56 @@ pub const GIGABYTE_PLUS_ONE: u64 = GIGABYTE + 1;
 pub const TERABYTE_PLUS_ONE: u64 = TERABYTE + 1;
```

Comment:
When I get usage I see this:
```
Uncompressed File Size: 19.289 MB
Server Utilization: 6.427 MB
Server Data Cap: 1000.000 KB
```

I think we should probably tweak the limits of the human duration stuff. I should never see 1000.000 KB.

We should see 1.000 MB (tweak the upper bounds here). But I should really see 1 MB (don't show trailing 0's).

You may encounter a lil bit of ugliness while doing this last step: https://stackoverflow.com/questions/59506403/how-to-format-a-float-without-trailing-zeros-in-rust